### PR TITLE
Run trace route how to

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-146ccd8994/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ac59cdf1df/how-to/)

**Comment by cyrichardson** [comment]
Here are some suggested edits: [run-a-trace-route_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1703761/run-a-trace-route_cyrEdits.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b4ac1e825e/how-to/).

File: content/rackspace-email/run-a-trace-route.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+type: article
+created_date: '2018-01-15'
+created_by: William Loy
+last_modified_date: '2018-02-07'
+last_modified_by: Nate Archer
+product: Rackspace Email
+product_url: rackspace-email
+--- 
+
+Trace routes help identify connectivity issues when you aretrying to connect to your Rackspace Email mailbox.
```
Comment (catlook) [inline content/rackspace-email/run-a-trace-route.md]: ... routes help to identify ...
... you are trying to ...

File: content/rackspace-email/run-a-trace-route.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+2. Open the **Terminal** application.
+
+    <img src="{% asset_path rackspace-email/how-to-run-a-trace-route/terminal.png %}"/>
+
+3. Enter the command ```traceroute secure.emailsrvr.com```.
+
+    <img src="{% asset_path rackspace-email/how-to-run-a-trace-route/secure_emailsrvr.png %}"/>
+
+   A list of time and server names displays.
```
Comment (catlook) [inline content/rackspace-email/run-a-trace-route.md]: ... of times and ...

File: content/rackspace-email/run-a-trace-route.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+#### Identifying issues
+
+A trace route shows the sequential server hop taken to reach a server, in addition to the time it took to complete each hop. When reviewing a trace route, you are typically looking for a delay or a failure in the trace progression.
```
Comment (catlook) [inline content/rackspace-email/run-a-trace-route.md]: ... server hops taken ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-19d6a896fc/how-to/).

**Comment by DonNateR** [comment]
@catlook: Should be good now.

**Pull request merged**